### PR TITLE
Fixed cyborgs using multi layer cables.

### PR DESCRIPTION
**Pull request by WarlockD**
## About The Pull Request
Fix for issue #51382

## Changelog
:cl:
fix: Cyborgs can now use multi layer cable interface
/:cl:

### CODE REVIEW

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -439,7 +439,7 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list(new/datum/stack_recipe("cable restrain
 	return(OXYLOSS)
 
 /obj/item/stack/cable_coil/proc/check_menu(mob/living/user)
-	if(!istype(user))
+	if(!istype(user) || !iscyborg(user))
```
Comment (ATH1909) [inline code/modules/power/cable.dm:442]: you sure about this one, chief?

won't this make this return false for anyone who ISN'T a cyborg?

Comment (WarlockD) [reply code/modules/power/cable.dm:442]: Humm yea your right I thought being human was already checked.

Comment (WarlockD) [reply code/modules/power/cable.dm:442]:
```
	if(!iscyborg(user) && !ishuman(user))
		return FALSE
```
This better? ishuman is a catch all for people who can work on wires?

Comment (WarlockD) [reply code/modules/power/cable.dm]: Wish it was like machinery where these checks are done before attack hand

**Comment by Jack7D1** [comment]
Thanks!

File: code/modules/power/cable.dm [hunk 2]
```diff
@@ -441,6 +441,8 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list(new/datum/stack_recipe("cable restrain
 /obj/item/stack/cable_coil/proc/check_menu(mob/living/user)
 	if(!istype(user))
 		return FALSE
+	if(!iscyborg(user) && !ishuman(user))
```
Comment (ShizCalev) [inline code/modules/power/cable.dm:444]: Why not a dexterity check?

File: code/modules/power/cable.dm [hunk 3]
```diff
@@ -724,6 +729,8 @@ GLOBAL_LIST(hub_radial_layer_list)
 /obj/structure/cable/multilayer/proc/check_menu(mob/living/user)
 	if(!istype(user))
 		return FALSE
+	if(!iscyborg(user) && !ishuman(user))
+		return FALSE
```
Comment (ShizCalev) [inline code/modules/power/cable.dm:733]: .

File: code/modules/power/cable.dm (hunk 2 above)
Comment (WarlockD) [reply code/modules/power/cable.dm:444]: Will a dex check work on borgs?

Comment (WarlockD) [reply code/modules/power/cable.dm]: Oh god.  I feel embarsed.  I don't know the mob code very well and didn't know about the 	IsAdvancedToolUser check.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**